annotations.md

```diff
@@ -29,7 +29,7 @@ This specification defines the following annotation keys, intended for but not l
   - version MAY be [Semantic versioning-compatible](https://semver.org/)
 - **org.opencontainers.image.revision** Source control revision identifier for the packaged software.
 - **org.opencontainers.image.vendor** Name of the distributing entity, organization or individual.
-- **org.opencontainers.image.licenses** License(s) under which contained software is distributed as an [SPDX License Expression][spdx-license-expression].
+- **org.opencontainers.image.licenses** License(s) under which contained software is distributed as an [SPDX License Expression](https://spdx.org/licenses/).
```

Comment:
Instead of updating this line, I think the URL at the end of the file should be updated.

Reply:
And the correct URL should be https://spdx.github.io/spdx-spec/v2.3/SPDX-license-expressions/